http-proxy-policy-operator/CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+<!-- Remember to update this file for your charm -- replace <charm-name> with the appropriate name. -->
+
+# Contributing
+
+## Overview
+
+This document explains the processes and practices recommended for contributing enhancements to the <charm-name> charm.
```

Comment:
I think the file should be updated for this charm.
e.g., `<charm-name>`

Reply:
Updated, thanks!